builtin/providers/azurerm/resource_arm_availability_set.go

```diff
@@ -93,6 +102,13 @@ func resourceArmAvailabilitySetCreate(d *schema.ResourceData, meta interface{})
 		Tags: expandTags(tags),
 	}
 
+	if managed == true {
+		n := "Aligned"
```

Comment:
According to the documentation here: https://docs.microsoft.com/en-us/rest/api/compute/availabilitysets/availabilitysets-create it's not a full sku object. It's just sku.name that is used.

sku.name	Specifies whether the availability set is managed or not. Possible values are: Aligned or Classic. An Aligned availability set is managed, Classic is not.

I thought the easier change was to specify managed or not as a boolean since that is how it's represented in the portal. Additionally in the sdk (https://github.com/Azure/azure-sdk-for-go/blob/master/arm/compute/models.go#L415) there is a property for "managed" but it does not appear in the REST docs and does not appear to be used when I tested it.